lib/prosopite.rb

```diff
@@ -227,6 +236,25 @@ def send_notifications
       raise NPlusOneQueriesError.new(notifications_str) if @raise
     end
 
+    def send_notifications
+      n_plus_one_errors = []
+      tc[:prosopite_notifications].each do |queries, kaller|
+        kaller = backtrace_cleaner.clean(kaller)
+        duration = tc[:prosopite_queries_duration][queries]
+        n_plus_one_errors << {
+          queries: queries,
+          stacktrace: kaller,
+          time: duration,
+        }
+      end
+
+      if @custom_logging_methods
```

Comment:
I believe we haven't tested this?

Reply:
Added tests [here](https://github.com/holistics/prosopite/pull/1/commits/38a0447ae5996d7f48c441469b3b2222194892a3)